api/Gamepad.json

```diff
@@ -0,0 +1,521 @@
+{
+  "version": "1.0.0",
+  "data": {
+    "api": {
+      "Gamepad": {
```

Comment:
I think we want a `__compat` object on the interface level here as it was prefixed in Chrome and Opera. 

Also, for Firefox, [there was a pref in the beginning](https://developer.mozilla.org/en-US/docs/Web/API/Gamepad_API/Using_the_Gamepad_API#Browser_compatibility). That need to be added here and elsewhere, too. (likely also applies to the other Gamepad files here)

```

{
  "version": "1.0.0",
  "data": {
    "api": {
      "Gamepad": {
        "__compat": {
          "basic_support": {
            "support": {
              "chrome": [
                {
                  "version_added": "35"
                },
                {
                  "prefix": "webkit",
                  "version_added": "21"
                }
              ],
              "edge": {
                "version_added": true
              },
              "firefox": [
                {
                  "version_added": "29"
                },
                {
                  "version_added": "24",
                  "version_removed": "28",
                  "flag": {
                    "type": "preference",
                    "name": "dom.gamepad.enabled",
                    "value_to_set": "true"
                  }
                }
              ],
              "ie": {
                "version_added": false
              },
              "opera": [
                {
                  "version_added": "22"
                },
                {
                  "prefix": "webkit",
                  "version_added": "15"
                }
              ],
              "safari": {
                "version_added": false
              },
              "webview_android": {
                "version_added": false
              },
              "firefox_android": {
                "version_added": "32"
              },
              "edge_mobile": {
                "version_added": true
              },
              "ie_mobile": {
                "version_added": false
              },
              "opera_android": {
                "version_added": false
              },
              "safari_ios": {
                "version_added": false
              },
              "chrome_android": {
                "version_added": true
              }
            }
          }
        }
      },
      "displayId": {
```